### ISSUE TITLE
Does not work for Firefox (Android)

### ISSUE THREAD

**Issue by edi1992** [xd4rker/MinerBlock#24]
Is it possible that the extension does not work for Firefox on Android? After installation coinhive.com and mineblock.org can load their mining scripts.

Can somebody confirm that?

**Comment by xd4rker** [comment]
Yes, MinerBlock is currently not compatible with Firefox for android. For some reason, it's still listed on FF for Android even though I have excluded its support :/

Anyway, you can use uBlock Origin for the time being. I will work on a compatible version soon.
